core/src/main/scala/io/finch/Filter.scala

```diff
@@ -0,0 +1,20 @@
+package io.finch
+
+import com.twitter.finagle.http.{Request, Response}
+
+trait Filter[F[_]] extends ((Request, EndpointCompiled[F]) => F[(Trace, Response)]) {
```

Comment:
I'm wondering if we want to change this name too. Not sure we want to have the naming collision with Finagle Filters.

Reply:
Yep, removed in favor of `Kleisli.compose`